crates/cli-support/src/js/binding.rs

```diff
@@ -819,12 +863,12 @@ fn instruction(
 
         Instruction::Retptr { size } => {
             js.cx.inject_stack_pointer_shim()?;
-            js.prelude(&format!(
-                "const retptr = wasm.__wbindgen_add_to_stack_pointer(-{});",
-                size
-            ));
+            let return_ptr = js.guarantee_alias(
+                "retptr",
+                format!("wasm.__wbindgen_add_to_stack_pointer(-{size})"),
+            );
```

Comment:
Any idea why this just has to be called `retptr`?

Reply:
Because `Instruction::LoadRetptr` has `retprt` hard-coded. `LoadRetptr` can't access the return pointer variable through the stack, so it's hard-coded.

I added a comment explaining this.